src/TelemetryChannels/ServerTelemetryChannel/Shared/Implementation/PartialSuccessTransmissionPolicy.cs

```diff
@@ -14,9 +14,9 @@
 
     internal class PartialSuccessTransmissionPolicy : TransmissionPolicy, IDisposable
     {
-        private readonly JavaScriptSerializer serializer = new JavaScriptSerializer();
+        private TaskTimer pauseTimer = new TaskTimer { Delay = TimeSpan.FromSeconds(TransmissionPolicyHelpers.SlotDelayInSeconds) };
 
-        private TaskTimer pauseTimer = new TaskTimer { Delay = TimeSpan.FromSeconds(SlotDelayInSeconds) };
+        public int ConsecutiveErrors { get; set; }
```

Comment:
It is reused. It keeps the count of consecutive errors so it can increase delay internal because we use exponential backoff. 

---
In reply to: [64499637](https://github.com/Microsoft/ApplicationInsights-dotnet/pull/232#discussion_r64499637) [](ancestors = 64499637)